src/hssm/config.py

```diff
@@ -41,6 +41,7 @@
             "default_priors": {
                 "v": {"name": "Uniform", "lower": -10.0, "upper": 10.0},
                 "a": {"name": "HalfNormal", "sigma": 2.0},
+                "z": None,
```

Comment:
Because `z` has bounds, a uniform prior will be automatically created. Specifying default priors here will actually truncate the default priors. This will be fixed when we support `inf` as bounds.

Reply:
ok sounds good.